experiments/process_sandbox/README.md

```diff
@@ -330,3 +329,184 @@ Code layout
+                                                                                                                                             
+  |-- 0x7f8000000000 [rw-s]                                                                                                                       
+  |                                                                                                                                          
+  |                                                                                                                                            |-- 0x7fbe80000000 [r--s]
+  |                                                                                                                                            |
+  ...                                                                                           |-- 0x7fbf00000000 [r--s]                      |
+  |                                                                                             |                                              |
+  |                                              |-- 0x7fbf80000000 [r--s]                      |                                              |
+  |                                              |                                              |                                              |        PageMap
+  |-- 0x7fc000000000                             |                                              |                                              |
+                                                 |                                              |        PageMap                               |
+                                                 |                                              |                                              |
+                                                 |        PageMap                               |                                              | 
+  |-- 0x7ffe80000000 -- 0x7ffec0000000 [rw-s]    |                                              |                                              |-- child 3 region [rw-s]
+                                                 |                                              |                                            
+  |-- 0x7fff00000000 -- 0x7fff40000000 [rw-s]    |                                              |-- child 2 region [rw-s]                     
+                                                 |                                                                                           
+  |-- 0x7fff80000000 -- 0x7fffc0000000 [rw-s]    |-- child 1 region [rw-s]                                                                   
+
+```
+
+The `library_runner` is responsible, at initialization time, for ensuring that both the `PageMap` and the child's memory region (i.e., heap) are correctly mapped in the process's address space.
+For that purposes, it uses the `memfd` file descriptors inherited from the parent.
```

Comment:
```suggestion
For that purposes, it uses the shared-memory file descriptors inherited from the parent.
```
It would be good to avoid Linux-specific descriptions as much as possible here.